src/Makefile

```diff
@@ -144,10 +144,18 @@ endif
 
 .PHONY:	clean host_clean all_platforms clang-format FORCE
 
+
+GIT_VERSION := $(shell if ! [ -x "$(command -v git)" ]; then git describe --always --dirty --tags; fi)
```

Comment:
If the error output from `git describe` is redirected via `2>/dev/null` then this should suppress the error line caused in the Make output. Alternatively you could add `-a -d .git` to the if [] clause to check that the .git directory exists as well as `git` being available

Reply:
If we check for .git then we don't really have to check for git binary being there do we?

Reply:
If, like us, one has a clone of BMD on a network accessible share, and tries doing a build from a machine other than the one that cloned the repo, that would result in there being a .git dir but not necessarily `git` being avaiable

Reply:
Ok, changed the logic to check for the .git directory only. This way it resolves the issue. If we have the .git directory we need to have git installed anyways so a double check is moot in this case.

Reply:
Ok, given that will result in a command-not-found when that's not the case, and that's on stderr iwrc, that works. We'll go with that